### PR TITLE
Sell media PayPal Upgradation

### PR DESCRIPTION
**Pull request by ashok-ladumor**
Paypal integration is done successfully with notification so users will know that they need to add API key, Also remove dead code

### CODE REVIEW

File: inc/admin-items.php [hunk 1]
```diff
@@ -726,3 +726,20 @@ function sell_media_save_bulk_edit() {
...
+*/
+add_filter( 'pre_get_posts', 'sell_media_show_loggedin_user_items' );
+function sell_media_show_loggedin_user_items($query) {
+
+	// get loggedin user data object
+	$_current_user_obj = wp_get_current_user();
+
+	if (is_admin() && in_array('contributor', $_current_user_obj->roles)) {
+
+		//Assing uset to media query
```
Comment (thadallender) [inline inc/admin-items.php:749]: Typo, please correct spelling.

File: inc/class-payments.php [hunk 2]
```diff
@@ -809,7 +809,7 @@ public function get_download_link( $payment_id = null, $product_id = null, $atta
 	 *
 	 * @since 0.1
 	 */
-	public function email_receipt( $payment_id = null, $email = null ) {
+	public function email_receipt( $payment_id = null, $email = null, $_comission = null ) {
```
Comment (thadallender) [inline inc/class-payments.php]: Typo - `commission` not `comission`

File: inc/class-payments.php [hunk 3]
```diff
@@ -818,13 +818,16 @@ public function email_receipt( $payment_id = null, $email = null ) {
 		$message['from_email'] = $settings->from_email;
 
 		// send admins and buyers different email subject and body
-		if ( $email == $message['from_email'] ) {
+		if ( $email == $message['from_email'] || $_comission != '') {
```
Comment (thadallender) [inline inc/class-payments.php]: Typo - `commission` not `comission`

File: inc/class-payments.php [hunk 4]
```diff
@@ -818,13 +818,16 @@ public function email_receipt( $payment_id = null, $email = null ) {
...
 
 			$message['subject'] = __( 'New sale notification', 'sell_media' );
 			$message['body']  = apply_filters( 'sell_media_email_admin_receipt_message_intro', '<p style="margin: 10px 0;">Congrats! You just made a sale!</p>' );
 			$message['body'] .= '<p style="margin: 10px 0;">' . __( 'Customer', 'sell_media' ) . ': ' . $this->get_buyer_name( $payment_id ) . '</p>';
 			$message['body'] .= '<p style="margin: 10px 0;">' . __( 'Address', 'sell_media' ) . ': ' . $this->get_buyer_address( $payment_id ) . '</p>';
 			$message['body'] .= '<p style="margin: 10px 0;">' . __( 'Email', 'sell_media' ) . ': ' . $this->get_meta_key( $payment_id, 'email' ) . '</p>';
+			if ($_comission) {
```
Comment (thadallender) [inline inc/class-payments.php]: Typo - `commission` not `comission`

File: inc/class-payments.php [hunk 5]
```diff
@@ -818,13 +818,16 @@ public function email_receipt( $payment_id = null, $email = null ) {
...
+				$message['body'] .= '<p style="margin: 10px 0;">' . __( 'Your Comission', 'sell_media' ) . ': $' . $_comission . '</p>';
```
Comment (thadallender) [inline inc/class-payments.php]: Typo - `commission` not `comission`

**Review by thadallender — CHANGES_REQUESTED** [review]

File: css/sell_media.css [hunk 6]
```diff
@@ -316,8 +316,12 @@ a:focus {
   margin-right: 10px;
 }
 
+
 /*= Buttons
 -------------------------------------------------------------- */
+.sell-media-dialog-box .sell-media-dialog-box-content a.sell-media-button {
```
Comment (thadallender) [inline css/sell_media.css:322]: Please make changes to sell_media.scss not sell_media.css. We compile sass to css.

**Review by thadallender — CHANGES_REQUESTED** [review]

File: inc/admin-items.php (hunk 1 above)
Comment (ashok-ladumor) [reply inc/admin-items.php]: Spelling corrected.

File: inc/class-payments.php (hunk 2 above)
Comment (ashok-ladumor) [reply inc/class-payments.php]: Changed

File: inc/class-payments.php (hunk 3 above)
Comment (ashok-ladumor) [reply inc/class-payments.php]: Changed

File: inc/class-payments.php (hunk 4 above)
Comment (ashok-ladumor) [reply inc/class-payments.php]: Changed

File: inc/class-payments.php (hunk 5 above)
Comment (ashok-ladumor) [reply inc/class-payments.php]: Changed

File: css/sell_media.css (hunk 6 above)
Comment (ashok-ladumor) [reply css/sell_media.css]: Sure, I have moved code from css to scss.

**Review by thadallender — APPROVED** [review]

**Pull request merged**